src/server/pachyderm_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"time"
 
 	"github.com/docker/go-units"
+	"github.com/gogo/protobuf/proto"
```

Comment:
hmm, did someone just mess up a merge before?

Reply:
Yeah master is broken again, i'll ask everyone to rebase